src/main/java/com/aquaticinformatics/aquarius/sdk/samples/create_service_models.sh

```diff
@@ -18,12 +18,13 @@ show_usage() {
 	exit_abort "$@"
 }
 
-Generator=../../../../../../../../../aquarius-sdk-net/src/SamplesServiceModelGenerator/bin/Release/net6.0/SamplesServiceModelGenerator.dll
+GeneratorProjectPath=../../../../../../../../../aquarius-sdk-net/src/SamplesServiceModelGenerator/SamplesServiceModelGenerator.csproj
+Generator=../../../../../../../../../aquarius-sdk-net/src/SamplesServiceModelGenerator/bin/Release/net8.0/SamplesServiceModelGenerator.dll
 
 ServerName=$1
 OutputPath=$2
 
```

Comment:
as per the comments in the .net pr, the line checking for the presence of `dotnet` would be useful.

it's probably also worth putting in a message that the java build relies on the presense of the aquarius-sdk-net repo, git cloned to the same directory as the aquarius-sdk-java repo on your local machine.